### ISSUE TITLE
Connection closed to http://supervisor/core

### ISSUE THREAD

**Issue by magicmatt007** [zachowj/node-red-contrib-home-assistant-websocket#736]
### Describe the bug

I understand this plugin is used for the Home Assistant Node Red Add-on.

Whilst this worked great on my Prod Home Assistant in the past, it now doesn't connect anymore to my home assistant:

In the logs I see an endless connecting / connected / connection closed loop.

I uninstalled the NodeRed AddOn, deleted the node-red folder in Hassio's config diretory and reinstalled the add on multiple times. No success. On other fresh installations of HA it still works...

Another user reported this also as an issue in the HA Node-red Add-on here: https://github.com/hassio-addons/addon-node-red/issues/1467

**Logs Node Red:**

...
4 Nov 17:16:34 - [info] [server:Home Assistant MS] Connecting to http://supervisor/core
4 Nov 17:16:34 - [info] [server:Home Assistant MS] Connected to http://supervisor/core
4 Nov 17:16:36 - [info] [server:Home Assistant MS] Connection closed to http://supervisor/core
4 Nov 17:16:36 - [info] [server:Home Assistant MS] Connection closed to http://supervisor/core
4 Nov 17:16:41 - [info] [server:Home Assistant MS] Connecting to http://supervisor/core
4 Nov 17:16:41 - [info] [server:Home Assistant MS] Connected to http://supervisor/core
4 Nov 17:16:42 - [info] [server:Home Assistant MS] Connection closed to http://supervisor/core
4 Nov 17:16:42 - [info] [server:Home Assistant MS] Connection closed to http://supervisor/core
...

**Logs Supervisor:**
...
22-11-04 17:28:40 INFO (MainThread) [supervisor.api.proxy] Home Assistant WebSocket API connection is closed
22-11-04 17:28:45 INFO (MainThread) [supervisor.api.proxy] Home Assistant WebSocket API request initialize
22-11-04 17:28:45 INFO (MainThread) [supervisor.api.proxy] WebSocket access from a0d7b954_nodered
22-11-04 17:28:45 INFO (MainThread) [supervisor.api.proxy] Home Assistant WebSocket API request running
22-11-04 17:28:46 INFO (MainThread) [supervisor.api.proxy] Home Assistant WebSocket API error: Received message 258:WebSocketError(<WSCloseCode.MESSAGE_TOO_BIG: 1009>, 'Message size 6323199 exceeds limit 4194304') is not str
...

### This package's version (not the Home Assistant add-on version)

don't know

### Is Node-RED running in Docker?

Home Assistant add-on

### Node-RED version

13.5.1

### Node.js version

don't know

### Additional context

The lastest version 13.5.1 of the Node Red Add-on is installed.
Home Assistant: 2022.11.0

**Comment by magicmatt007** [comment]
I solved the problem for my installation:
The "problem" is that ALL entities are sent in a single websocket call.

* I hence looked at my entities under "Developer Tools/States". I had A LOT of device tracker entities with an unknown state. Probably left overs from a previously used BLE tracker.
* I then went to my /config directory, and looked at "known_devices" file. It was blown up to 3.8kb with A LOT of entries. As I don't really need the know_devices file I made a copy of it and deleted the original file.
* After a restart of HA the error was gone, and Node Red worked fine again
